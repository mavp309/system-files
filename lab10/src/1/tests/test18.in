i 5
i 15
i 25
s 15
r -1
p -1
d 5
p -1
q 0
