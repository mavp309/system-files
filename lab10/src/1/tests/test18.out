True
5 15 25 
Deleted
15 25 
